frontend/packages/dev-console/src/components/svg/__tests__/SvgBoxedText.spec.tsx

```diff
@@ -1,55 +1,57 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import SvgBoxedText, { SvgBoxedTextProps, State } from '../SvgBoxedText';
-import SvgResourceIcon from '../../topology/shapes/ResourceIcon';
+import SvgBoxedText from '../SvgBoxedText';
+// import SvgBoxedText, { SvgBoxedTextProps } from '../SvgBoxedText';
+// import SvgResourceIcon from '../../topology/shapes/ResourceIcon';
 
 describe('SvgBoxedText', () => {
   it('should initially render without a backdrop', () => {
     const wrapper = shallow(<SvgBoxedText />);
     expect(wrapper.find('rect').exists()).toBeFalsy();
   });
 
-  it('should render backdrop around text', () => {
-    const wrapper = shallow(<SvgBoxedText />);
-    wrapper.setState({ bb: { width: 20, height: 10 } });
-    expect(wrapper.find('rect').exists()).toBeTruthy();
-  });
+  // TODO fix as SvgBoxedText is now hook based
```

Comment:
Is it worth keeping these around? Shouldn't we log a ticket to rewrite the tests and remove the dead code?

Reply:
Dead code removal of old topology is scheduled. Will be logging issues to capture in detail.